### ISSUE TITLE
New release causes error on import

### ISSUE THREAD

**Issue by mtreinish** [datadriventests/ddt#83]
When running test suites locally with the new 1.4.0 release a traceback is raised on import because of the use `nose`. For example:

```
Failed to import test module: test.python
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.7/x64/lib/python3.7/unittest/loader.py", line 470, in _find_test_path
    package = self._get_module_from_name(name)
  File "/opt/hostedtoolcache/Python/3.7.7/x64/lib/python3.7/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/vsts/work/1/s/test/__init__.py", line 17, in <module>
    from ddt import data, unpack
  File "/opt/hostedtoolcache/Python/3.7.7/x64/lib/python3.7/site-packages/ddt.py", line 15, in <module>
    from nose.tools import nottest
ModuleNotFoundError: No module named 'nose'
```

This is in a stdlib unittest test suite. `nose` is not listed in the requirements list for the new ddt package on pypi so this causes a failure since nose will not necessarily be installed. That being said nose is an inactive project and the first section in their docs suggest not using it: https://nose.readthedocs.io/en/latest/#note-to-users so it might be better to remove this import.

**Comment by ppentchev** [comment]
Hi,

In addition to nose, ddt also seems to require enum now, which means it needs a dependency on enum34 on Python 2.7. See https://github.com/datadriventests/ddt/pull/85

G'luck,
Peter

**Comment by wswld** [comment]
Hi, 

I see the problem, but reverting the whole commit (with the introduced functionality) is a bit abrupt. Let me look how it can be fixed less radically. 

What for the enum the tests for 2.7 are somehow still running, I'm not sure how I didn't notice that.

**Comment by ppentchev** [comment]
Hi,

The tests probably ran because flake8 brings in enum34 as an installation dependency :) But `python3 setup.py bdist_wheel` and then, in a pristine virtual environment, `python2 -m pip install ddt-1.4.0-*.whl` will demonstrate the missing dependencies.

Thanks for looking into this!

G'luck,
Peter

**Comment by wswld** [comment]
The issue should be fixed in 1.4.1
